rotkehlchen/db/evmtx.py

```diff
@@ -585,25 +585,25 @@ def _form_evm_transaction_dbquery(self, query: str, bindings: list[Any], has_pre
                 'SELECT DISTINCT evm_transactions.tx_hash, evm_transactions.chain_id, timestamp, block_number, from_address, to_address, value, gas, gas_price, gas_used, input_data, nonce FROM evm_transactions ' + query,  # noqa: E501
                 bindings,
             )
-        else:
-            return (
-                'SELECT DISTINCT evm_transactions.tx_hash, evm_transactions.chain_id, timestamp, block_number, from_address, to_address, value, gas, gas_price, gas_used, input_data, nonce FROM (SELECT * from evm_transactions ORDER BY timestamp DESC LIMIT ?) AS evm_transactions ' + query,  # noqa: E501
-                [FREE_ETH_TX_LIMIT] + bindings,
-            )
+        # else
+        return (
+            'SELECT DISTINCT evm_transactions.tx_hash, evm_transactions.chain_id, timestamp, block_number, from_address, to_address, value, gas, gas_price, gas_used, input_data, nonce FROM (SELECT * from evm_transactions ORDER BY timestamp DESC LIMIT ?) AS evm_transactions ' + query,  # noqa: E501
+            [FREE_ETH_TX_LIMIT] + bindings,
+        )
 
-    def _build_evm_transaction(self, result: tuple[Union[str, int]]) -> EvmTransaction:
+    def _build_evm_transaction(self, result: tuple[Any, ...]) -> EvmTransaction:
```

Comment:
Since we know the fields... isn't it worth to have the tuple type with the type for each field and use it here?

Reply:
> Since we know the fields... isn't it worth to have the tuple type with the type for each field and use it here?

no it really isn't. Would need to create a new special tuple type for this just to typecheck this one call. And it's meaningless since cursor.execute() returns tuple[Any, ...]